dbms/src/Functions/tests/gtest_cast_json_as_string.cpp

```diff
@@ -120,7 +120,7 @@ try
     /// ColumnConst(nullable)
     auto nested_str_col = ColumnString::create();
     nested_str_col->insertData(reinterpret_cast<const char *>(bj9), sizeof(bj9) / sizeof(UInt8));
-    col_null_map = ColumnUInt8::create(3, 0);
+    col_null_map = ColumnUInt8::create(1, 0);
     json_col = ColumnNullable::create(std::move(nested_str_col), std::move(col_null_map));
```

Comment:
Add an assert check that `nested_column->size() == null_map->size()` in `ColumnNullable::ColumnNullable(MutableColumnPtr && nested_column_, MutableColumnPtr && null_map_)` is better